addons/l10n_rs_edi/models/account_move.py

```diff
@@ -0,0 +1,157 @@
+import uuid
+import requests
+import json
+
+from odoo import _, api, fields, models
+from requests.exceptions import Timeout, ConnectionError, HTTPError
+
+DEMO_EFAKTURA_URL = 'https://demoefaktura.mfin.gov.rs/api/publicApi/sales-invoice/ubl'
+EFAKTURA_URL = 'https://efakturadev.mfin.gov.rs/api/publicApi/sales-invoice/ubl'
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_rs_edi_uuid = fields.Char("Invoice UUID", copy=False, help="Unique Identifier for an invoice used as request id")
+
+    l10n_rs_edi_is_eligible = fields.Boolean(
+        compute="_compute_l10n_rs_edi_is_eligible",
+        help="Serbia: technical field to determine if this invoice is eligible to be e-invoiced.",
+    )
+
+    l10n_rs_edi_attachment_file = fields.Binary(
+        string="Serbian E-Invoice XML File",
+        copy=False,
+        attachment=True,
+        help="Serbia: technical field holding the e-invoice XML data.",
+    )
+
+    l10n_rs_edi_attachment_id = fields.Many2one(
+        comodel_name='ir.attachment',
+        string="eFaktura XML Attachment",
+        compute=lambda self: self._compute_linked_attachment_id('l10n_rs_edi_attachment_id', 'l10n_rs_edi_attachment_file'),
+        depends=['l10n_rs_edi_attachment_file'],
+    )
+
+    l10n_rs_edi_state = fields.Selection(
+        string="Serbia E-Invoice state",
+        selection=[
+            ('to_send', 'Sending'),
+            ('sent', 'sent'),
+            ('sending_failed', 'Error')
+        ],
+        tracking=True,
+        readonly=True,
+        copy=False,
+    )
+
+    l10n_rs_edi_error = fields.Text(
+        string="Serbia E-Invoice error",
+        copy=False,
+        readonly=True,
+    )
+
+    l10n_rs_tax_date_obligations_code = fields.Selection(
+        string="Tax Date Obligations",
+        selection=[
+            ('35', 'By Delivery Date'),
+            ('3', 'By Issuance Date'),
+            ('432', 'By Billing System'),
+        ],
+        default='3',
+    )
+    l10n_rs_edi_invoice_id = fields.Char("Invoice Id", copy=False)
+    l10n_rs_edi_sales_invoice_id = fields.Char("Sales Invoice Id", copy=False)
+    l10n_rs_edi_purchase_invoice_id = fields.Char("Purchase Invoice Id", copy=False)
+
+    @api.depends('country_code', 'move_type')
+    def _compute_show_delivery_date(self):
+        # EXTENDS 'account'
+        super()._compute_show_delivery_date()
+        for move in self:
+            if move.country_code == 'RS':
+                move.show_delivery_date = move.is_sale_document()
+
+    @api.depends("country_code", "move_type")
+    def _compute_l10n_rs_edi_is_eligible(self):
+        for move in self:
+            move.l10n_rs_edi_is_eligible = move.country_code == 'RS' and move.is_sale_document()
+
+    def _post(self, soft=True):
+        posted = super()._post(soft)
+        posted.l10n_rs_edi_state = 'to_send'
```

Comment:
No!!! It must be send by the send and print wizard if you choose to send it. Also, it should be sent directly from the wizard so this value is useless imo